src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types';
+import { getUserById } from '../../services';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
Just keep a user in todo. `TodoInfo` is a presentational component, it shouldn't have any logic
You can read about presentational and container components https://www.geeksforgeeks.org/reactjs-container-and-presentational-pattern-in-components/